### DISCUSSION TITLE
CSS variables in the source code become SCSS in npmjs

### DISCUSSION THREAD

**Discussion by okadurin** [twbs/bootstrap#36033] [General] [answered]
Hi, I noticed that there are CSS variables in the source code which become SCSS variables in production.
Here is an example in the [source code](https://github.com/twbs/bootstrap/blob/33827d24a4f6a4c8446584d7740c338e35b41b91/scss/_variables.scss#L1432) :
`$alert-border-radius:           var(--#{$prefix}border-radius) !default;`
And here when getting bootstrap via `npm install`
`$alert-border-radius: $border-radius !default;`

May I ask you what program does this transformation and why it happens? Would appreciate any references to the doc!

**Comment by WinterSilence** [comment, marked as answer]
you use install/build same version?

**Comment by okadurin** [comment]
@WinterSilence Thank! You are right. I checkout out 5.1.3 now in source code and compared it in the one from `npmjs`. Those are the same. And before that I was comparing `main` and 5.1.3 from `mpmjs`
